PracticeGit/Complex.c

```diff
@@ -0,0 +1,40 @@
+// ------------------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// ------------------------------------------------------------------------------------------------
+// Program.c - Functions for various arithmetic operations on complex numbers.
+// ------------------------------------------------------------------------------------------------
+#include "Complex.h"
+#include <math.h>
+
+ComplexNumber OpAdd (ComplexNumber a, ComplexNumber b) {
+   ComplexNumber c;
+   c.real = a.real + b.real;
+   c.img = a.img + b.img;
```

Comment:
Why not use the following syntax to declare structs? It is more elegant.
 ComplexNumber c = { a, b };